src/hooks/useMintPoints.tsx

```diff
@@ -0,0 +1,41 @@
+import { Address } from "viem"
+
```

Comment:
what is the purpose of this `empty line` between imports?
- can we remove it? 🚮 
```suggestion
```